<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GonioControl</class>
 <widget class="QMainWindow" name="GonioControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(177, 177, 177);
background-color: rgb(24, 24, 24);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

background-color: rgb(0, 212, 116);
background-color: rgb(24, 24, 24);
</string>
    </property>
    <property name="title">
     <string>X / Y Move</string>
    </property>
    <widget class="QToolButton" name="up">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(24, 24, 24);</string>
     </property>
     <property name="text">
      <string>up</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>SHIFTUP3.bmp</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QToolButton" name="down">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(24, 24, 24);</string>
     </property>
     <property name="text">
      <string>down</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>SHIFTDOWN3.bmp</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QToolButton" name="right">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(24, 24, 24);</string>
     </property>
     <property name="text">
      <string>right</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>SHIFTRIGHT3.bmp</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QToolButton" name="left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(24, 24, 24);</string>
     </property>
     <property name="text">
      <string>left</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>SHIFTLEFT3.bmp</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Xvalue">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1200.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1200.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Yvalue">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1200.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1200.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Step:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="StepShift">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="singleStep">
      <double>0.020000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QToolButton" name="NSHIFT">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(9, 163, 85);</string>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tilt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Tilt X</string>
    </property>
    <widget class="QToolButton" name="Txplus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>TILT3.bmp</normaloff>TILT3.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="Txmoins">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>TILT3-.bmp</normaloff>TILT3-.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="TXvalue">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-70.000000000000000</double>
     </property>
     <property name="maximum">
      <double>70.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>2.000000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Step:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="StepTiltX">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QToolButton" name="NTX">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(9, 163, 85);</string>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
    <zorder>Txmoins</zorder>
    <zorder>Txplus</zorder>
    <zorder>TXvalue</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>StepTiltX</zorder>
    <zorder>NTX</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Z</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QToolButton" name="Zplus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>ZGONIOblack+.bmp</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Zvalue">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>80</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-220.000000000000000</double>
     </property>
     <property name="maximum">
      <double>220.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>3.000000000000000</double>
     </property>
    </widget>
    <widget class="QToolButton" name="Zmoins">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>ZGONIOblack-.bmp</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Step:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="StepZ">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QToolButton" name="NZ">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(9, 163, 85);</string>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="RefreshButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(177, 177, 177);</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Neutralize">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="text">
     <string>Neutral</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="tilt_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Tilt Y</string>
    </property>
    <widget class="QToolButton" name="Typlus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>TILT3.bmp</normaloff>TILT3.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="Tymoins">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>TILT3-.bmp</normaloff>TILT3-.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="TYvalue">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-25.000000000000000</double>
     </property>
     <property name="maximum">
      <double>25.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>2.000000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Step:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="StepTiltY">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QToolButton" name="NTY">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(9, 163, 85);
border-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
